files/lshell.conf

```diff
@@ -22,7 +22,7 @@ forbidden       : [';','&','`','>','<','$(','${']
 warning_counter : -1
 
 ##  command aliases list (similar to bash’s alias directive)
-aliases         : {'ll':'ls -l', 'vim':'rvim'}
+aliases         : {'ll':'ls -l', 'vim':'rvim', 'kubectl':'/snap/bin/kubectl', 'postdeploy':'/home/ubuntu/postdeploy'}
```

Comment:
Can we put postdeploy in /home/ubuntu/bin and add that to the path if not already there?
If it's trouble, nevermind.

Reply:
Done.